### PR TITLE
Json data file

### PR DESCRIPTION
**Pull request by DaScOrPiO**

### CODE REVIEW

File: src/app/data/data.json [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+            "link": "",
+            "name": ""
+        },
+        "image": "",
+        "tags": [],
+        "metadata": {
+            "twitter": "",
+            "website": ""
+        },
+        "languages": ["Typescript", "Docker"]
```
Comment (Extheoisah) [inline src/app/data/data.json:67]: Lets stick with one convention of writing the languages. Also, Docker is not a language
```suggestion
        "languages": ["TS"]
```

File: src/app/data/data.json [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+            "link": "",
+            "name": ""
+        },
+        "image": "",
+        "tags": [],
+        "metadata": {
+            "twitter": "",
+            "website": ""
+        },
+        "languages": ["Js"]
```
Comment (Extheoisah) [inline src/app/data/data.json:101]:
```suggestion
        "languages": ["JS"]
```

File: src/app/data/data.json [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+            "twitter": "",
+            "website": ""
+        },
+        "languages": ["C++", "Python", "C"]
+    },
+
+    "Bitcoin XPUB Converter": {
+        "name": "Bitcoin XPUB Converter",
+        "description": "",
+        "link": "https://github.com/jlopp/xpub-converter",
```
Comment (Extheoisah) [inline src/app/data/data.json:90]:
```suggestion
        "link": "https://github.com/jlopp/xpub-converter/issues",
```

**Review by Extheoisah — CHANGES_REQUESTED** [review]
looks good. 
Just a few changes. Make sure to add `/issues` to all repo links.

File: src/app/data/data.json (hunk 1 above)
Comment (DaScOrPiO) [reply src/app/data/data.json:67]: Alright

File: src/app/data/data.json (hunk 3 above)
Comment (DaScOrPiO) [reply src/app/data/data.json:90]: Noted

File: src/app/data/data.json (hunk 2 above)
Comment (DaScOrPiO) [reply src/app/data/data.json:101]: Noted

**Review by Extheoisah — APPROVED** [review]
All good!

**Pull request merged**